sb/robot/vision.py

```diff
@@ -0,0 +1,20 @@
+from collections import namedtuple
+
+# Points
+# TODO: World Coordinates
+PolarCoord = namedtuple("PolarCoord", "distance_meters rot_y_rad rot_y_deg")
+CartCoord = namedtuple("CartCoord", "x y z")
+
+# Marker class
+MarkerBase = namedtuple("Marker",
+                        "id size polar pixel_centre cartesian is_wall_marker is_token_marker")
+
+
+class Marker(MarkerBase):
+    def __new__(cls, *args, **kwargs):
```

Comment:
This function call is unnecessary

Reply:
Fixed in https://github.com/sourcebots/simulator/pull/1/commits/afb03723d0c28a26ec66a6f5ed6fa5c3c646f7b6